cspell/.cspell.json

```diff
@@ -985,6 +986,7 @@
         "unsqueeze",
         "URDF",
         "Uref",
+        "Urefex",
```

Comment:
I think `transed` and `Urefex` shouldn't be added because they can be replaced by `transformed` and `Uref_ex`.

Reply:
ok, i'll fix the codes which is using them instead